audinterface/core/segment.py

```diff
@@ -338,21 +330,15 @@ def process_files(
             ends=ends,
             root=root,
         )
+        if len(series) == 0:
+            return audformat.filewise_index()
```

Comment:
I had to add this for `audformat>=0.15.0` as `audformat.utils.union([])` now returns:
```
Index([], dtype='object')
```